metadata-load/src/main/resources/application.properties

```diff
@@ -8,3 +8,23 @@ ena.datasource.tomcat.maxActive=2
 
 # Analysis SQL query
 ena.analysis.query=SELECT ANALYSIS_XML FROM ERA.ANALYSIS WHERE ANALYSIS_ID = :accession
+
+
+spring.application.name=metadata-import
```

Comment:
Make this the first property in the file. Why does it need a name though?

Reply:
just as we run it as a separate application jar I gave it a name.